core/arch/arm/mm/mobj.c

```diff
@@ -155,12 +157,6 @@ struct mobj *mobj_phys_alloc(paddr_t pa, size_t size, uint32_t cattr,
 	enum teecore_memtypes area_type;
 	void *va;
 
-	if ((pa & CORE_MMU_USER_PARAM_MASK) ||
```

Comment:
Now that you mention it, I don't see why it should be removed. From what I've seen so far is the dynamic shared memory stuff depending on `CFG_SMALL_PAGE_USER_TA` which is also defining `CORE_MMU_USER_PARAM_SHIFT` as `SMALL_PAGE_SHIFT`.